app/controllers/integrations/base_controller.rb

```diff
@@ -60,4 +55,25 @@ def service_type
   def service_name
     @service_name ||= self.class.name.demodulize.sub('Controller', '').downcase
   end
+
+  def create_docker_build
+    release = create_new_release
```

Comment:
`create_new_release || latest_release` would avoid making create_release also do a find
or rename to `create_or_find_release`